docs/admin/Import-and-Update.md

```diff
@@ -189,19 +189,25 @@ entire US adds about 10GB to your database.
   1. Get preprocessed TIGER 2018 data and unpack it into the
      data directory in your Nominatim sources:
 
-        cd Nominatim/data
-        wget https://nominatim.org/data/tiger2018-nominatim-preprocessed.tar.gz
-        tar xf tiger2018-nominatim-preprocessed.tar.gz
+```sh
+    cd Nominatim/data
+    wget https://nominatim.org/data/tiger2018-nominatim-preprocessed.tar.gz
+    tar xf tiger2018-nominatim-preprocessed.tar.gz
+```
 
     `data-source/us-tiger/README.md` explains how the data got preprocessed.
 
-  2. Import the data into your Nominatim database: 
+  2. Import the data into your Nominatim database:
 
-        ./utils/setup.php --import-tiger-data
+```sh
+    ./utils/setup.php --import-tiger-data
+```
 
   3. Enable use of the Tiger data in your `settings/local.php` by adding:
 
+```
          @define('CONST_Use_US_Tiger_Data', true);
+```
```

Comment:
Intermixing list items and code blocks does not really work as expected. We probably should get rid of the list completely to get this right. Something for another change tough. Do you mind removing those for now? The rest looks good to merge.

Reply:
I have reverted those changes for now.